### PR TITLE
Add ppocr series models

### PR DESCRIPTION
**Pull request by VictorHe-1**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

1. 适配ppocr系列模型，增加ppocr系列模型的推理配置文件，以及进行模型注册。

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: deploy/py_infer/src/configs/rec/ppocr/ch_PP-OCRv2_rec_distillation.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  use_space_char: True
+
+eval:
+  dataset:
+    transform_pipeline:
+      - DecodeImage:
+          img_mode: BGR
+          channel_first: False
+          to_float32: True
+      - CTCLabelEncode:
```
Comment (liangxhao) [inline deploy/py_infer/src/configs/rec/ppocr/ch_PP-OCRv2_rec_distillation.yaml:12]: CTCLabelEncode在预处理里面用不到吧？

File: deploy/py_infer/src/configs/det/ppocr/ch_PP-OCRv2_det_cml.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    transform_pipeline:
+      - DecodeImage:
+          img_mode: BGR
+          channel_first: False
+      - ResizeImage:
+      - NormalizeImage:
+          mean: imagenet
+          std: imagenet
+          is_hwc: True
+      - ToCHWImage:
```
Comment (liangxhao) [inline deploy/py_infer/src/configs/det/ppocr/ch_PP-OCRv2_det_cml.yaml:20]: 文件末尾要有换行符

File: deploy/py_infer/src/configs/det/ppocr/ch_det_mv3_db_v2.0.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  expand_ratio: 1.5
+
+
+eval:
+  dataset:
+    transform_pipeline:
+      - DecodeImage:
+          img_mode: BGR
+          channel_first: False
+      - ResizeImage:
```
Comment (liangxhao) [inline deploy/py_infer/src/configs/det/ppocr/ch_det_mv3_db_v2.0.yaml:15]: 映射为keep_ratio=True的Resize，即ScalePadImage

File: deploy/py_infer/src/configs/rec/ppocr/ch_PP-OCRv2_rec_distillation.yaml (hunk 1 above)
Comment (VictorHe-1) [reply deploy/py_infer/src/configs/rec/ppocr/ch_PP-OCRv2_rec_distillation.yaml:12]: 是的，增加在preprocess_skip_ops里面了，只是增加个示例

File: deploy/py_infer/src/configs/det/ppocr/ch_PP-OCRv2_det_cml.yaml (hunk 2 above)
Comment (VictorHe-1) [reply deploy/py_infer/src/configs/det/ppocr/ch_PP-OCRv2_det_cml.yaml:20]: 好的

File: deploy/py_infer/src/configs/det/ppocr/ch_det_mv3_db_v2.0.yaml (hunk 3 above)
Comment (VictorHe-1) [reply deploy/py_infer/src/configs/det/ppocr/ch_det_mv3_db_v2.0.yaml:15]: ok

**Review by liangxhao — APPROVED** [review]
ok

**Comment by VictorHe-1** [comment]
> 

ok

**Review by HaoyangLee — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]
存疑点：server / mobile naming

**Pull request merged**